<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewFrames</class>
 <widget class="QDialog" name="NewFrames">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>286</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>385</width>
    <height>286</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>385</width>
    <height>286</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AvisionR - New Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(255, 255, 255);
background-color: #262626;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Price: $</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Price">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #686868;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QCommandLinkButton" name="commandLinkButton_add">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="commandLinkButton_cancel">
       <property name="maximumSize">
        <size>
         <width>179</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Units:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_units"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Brand:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Brand">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #686868;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Model:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Model">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #686868;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Color:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Color">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #686868;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Brand</tabstop>
  <tabstop>lineEdit_Model</tabstop>
  <tabstop>lineEdit_Color</tabstop>
  <tabstop>lineEdit_Price</tabstop>
  <tabstop>spinBox_units</tabstop>
  <tabstop>commandLinkButton_add</tabstop>
  <tabstop>commandLinkButton_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
